clikt/src/commonMain/kotlin/com/github/ajalt/clikt/parameters/options/FlagOption.kt

```diff
@@ -79,8 +80,16 @@ inline fun <OutT> OptionWithValues<Boolean, Boolean, Boolean>.convert(
 /**
  * Turn an option into a flag that counts the number of times it occurs on the command line.
  */
-fun RawOption.counted(): OptionWithValues<Int, Int, Int> {
-    return int().transformValues(0..0) { it.lastOrNull() ?: 1 }.transformAll { it.sum() }
+@JvmOverloads // TODO(5.0): remove this annotation
+fun RawOption.counted(limit: Int = 0, clamp: Boolean = true): OptionWithValues<Int, Int, Int> {
+    return int().transformValues(0..0) { it.lastOrNull() ?: 1 }.transformAll {
```

Comment:
```suggestion
    require(limit > 0) { "limit must be > 0: $limit" }
    return int().transformValues(0..0) { it.lastOrNull() ?: 1 }.transformAll {
```
probably also want a test for that

(or `require(limit >= 0)` if you switch to `Int.MAX_VALUE` and allowing 0)